sdk/keyvault/azure-security-keyvault-keys/src/test/java/com/azure/security/keyvault/keys/KeyAsyncClientTest.java

```diff
@@ -80,6 +80,16 @@ public void createRsaKey(HttpClient httpClient, KeyServiceVersion serviceVersion
             .verifyComplete());
     }
 
+    /**
+     * Tests that a Ec key created.
+     */
+    @ParameterizedTest(name = DISPLAY_NAME_WITH_ARGUMENTS)
+    @MethodSource("getTestParameters")
+    public void createEcKey(HttpClient httpClient, KeyServiceVersion keyServiceVersion) {
+        createKeyAsyncClient(httpClient, keyServiceVersion);
+        createEcKeyRunner((expected) -> assertNotNull(client.createEcKey(expected)));
```

Comment:
I think we can use `assertKeyEquals` like the method above to verify the created key contains the same characteristics we sent in the payload.

```suggestion
        createEcKeyRunner((expected) -> StepVerifier.create(client.createEcKey(expected))
            .assertNext(response -> assertKeyEquals(expected, response))
            .verifyComplete());
```